tools/swift-machine-learning-api/README.md

```diff
@@ -0,0 +1,336 @@
+    |__ update.sh  (bash script for updating trainer package) 
+    |__ config
+        |__ train.yaml  (yaml file for train configurations)
+        |__ config_file.yaml  (yaml file for API configurations)
+        |__ developer.yaml  (yaml file for developer configurations)
+    |__ codes                   
+        |__ config.yaml  (yaml file for CloudML configurations)
+        |__ setup.py 
+        |__ trainer  (trainer package)
+            |__ __init__.py
+            |__ launch_demo.py  (end-to-end script for running package)
+            |__ models.py  (tensorflow estimator functions)
+            |__ input_pipeline_dask.py  (handles input_pipeline)
+            |__ utils
+                |__ __init__.py
+                |__ custom_utils_fn.py
+                |__ optimizer_utils.py
+                |__ metric_utils.py
+```
+
+## Instructions to run the code
+
+### Configuring the API
+
+- Create a config file named **_'config_file.yaml'_** like below, with your project id, bucket name and path of service account key along for main.py in the **config directory** for interacting with GCP resources.
+
+  ```
+  bucket_name: "gs://<bucket-name>"
+  project: <project_name>
+  project_id: <project_ID>
+  service_account_json_key: <path/of/service/account/key/json/file>
+  ```
+
+- Create a config file named **_'train.yaml'_** like below along for train.py in the **config directory** to specify train configurations. Refer to [*CMLE machine types*](https://cloud.google.com/ml-engine/docs/tensorflow/machine-types) for specifying type virtual machines you want the model to be trained. The trainer package will be created and updated in GCS bucket by running bash script update.sh. 
+
+    ```
+    scaleTier: "CUSTOM"
+    masterType: "complex_model_m"
+    workerType: "complex_model_m"
+    parameterServerType: "large_model"
+    workerCount: 3
+    parameterServerCount: 1
+    packageUris: ['<path/of/trainer/package/in/GCS>'] # bucket_name+'/trainer-0.0.tar.gz' 
+    region: "us-central1"
+    jobDir: <path/of/the/job_dir>
+    runtimeVersion: <tensorflow version>
+    pythonVersion: <python version> ##python2.7
+    ```
+- Create a config file named **_'config.yaml'_** like below in **codes** folder, with the type of virtual machines you want the model to be trained on. Currently if the training job is submitted through Train API the machine configurations are directly loaded from train.yaml. However to this has been added to provide an option to launch training jobs using [*command line*](https://cloud.google.com/sdk/gcloud/reference/ml-engine/jobs/submit/training)
+Example of  **_'config.yaml'_** looks like 
+  ```
+  trainingInput:
+    scaleTier: CUSTOM
+    masterType: complex_model_l
+    workerType: complex_model_l
+    parameterServerType: large_model
+    workerCount: 3
+    parameterServerCount: 1
+  ```
+
+- Check that your Storage and Machine Learning Engine API is enabled.
```

Comment:
Add gcloud command for enabling services.

Reply:
Added.